.circleci/config.yml

```diff
@@ -1,8 +1,8 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@5
-  slack: circleci/slack@4.4.2
+  hmpps: ministryofjustice/hmpps@6
+  slack: circleci/slack@4.10
```

Comment:
Thought I'd remove the bug fix part, probably always want them to be updated automagically. Assuming CircleCI slack orb is pretty safe and secure.

Wondering whether we should just have it at v4? Save us updating it.

Reply:
In fact, is the slack orb even used? Can't see any references to **slack/** in the config? 🤔 Maybe it's been abstracted away by the hmpps-orb, and it's a dangling bit of tech-debt?

Reply:
Looks like the slack functionality was removed by me when taking out the **check-outdated** job. I'll remove the orb on this PR as well 👍